pkg/vsphere/vm/vm.go

```diff
@@ -303,6 +303,65 @@ func (vm *VirtualMachine) WaitForKeyInExtraConfig(ctx context.Context, key strin
 	return detail, nil
 }
 
+// WaitForKeyChange will block until it detects a change in value for ANY of the provided key value combinations. Additionally it will unblock if the container has powered off, returning an error in such a case.
+func (vm *VirtualMachine) WaitForKeyChange(op trace.Operation, keys map[string]string) error {
+	var poweredOff error
+
+	// fix up the filter map. value of "" ---> "<nil>"
```

Comment:
You can remove this comment - it's clear that we're escaping the value from the function name.